src/components/Trilemma/index.tsx

```diff
@@ -32,55 +32,46 @@ const Trilemma = () => {
     ...trilemmaChecks,
   }
 
-  // TODO: Replace with direct token implementation after UI migration is completed
-  const lgBp = useToken("breakpoints", "lg")
-
   return (
     <Flex
-      alignItems="flex-start"
+      alignItems="center"
       flexDirection={{ base: "column", lg: "row" }}
       justifyContent="space-between"
+      columnGap="12"
+      rowGap="8"
     >
-      <Flex
-        flexDirection="column"
-        my={{ base: 8, md: 12 }}
-        mx={{ md: 12 }}
-        py={8}
+      <Stack
+        spacing="8"
+        mt={{ base: "16", md: "20" }}
+        mb={{ lg: "20" }}
+        ms={{ md: "12" }}
+        me={{ md: "12", lg: 0 }}
         flex={{
-          base: "1 1 100%",
           lg: "0 1 500px",
         }}
       >
-        <OldHeading fontSize="2rem" mt={0}>
+        <Heading fontSize="2rem">
```

Comment:
@nloureiro Brought this up elsewhere, but more use of a non-standard font size

<img width="306" alt="image" src="https://github.com/ethereum/ethereum-org-website/assets/54227730/da631754-387c-4593-b803-14f107917dbc">

^ These are the sizes we have assigned to tokens right now... not requesting anything for this PR, just noting... [card here](https://www.notion.so/efdn/Review-use-of-non-standard-font-sizes-713321312cd04b0cbd8d28eb37ebfe5b?pvs=4)